enterprise/server/auth/BUILD

```diff
@@ -6,7 +6,6 @@ go_library(
     importpath = "github.com/buildbuddy-io/buildbuddy/enterprise/server/auth",
     visibility = [
         "//enterprise:__subpackages__",
```

Comment:
There are a few changes in here where we remove `"@buildbuddy_internal//enterprise:__subpackages__",` but still leave the visibility = ["//enterprise:__subpackages__",] attribute set in a target. Can those also be moved to a `package(default_visibility = ["//enterprise:__subpackages__"])` rule instead?

Reply:
Yup, definitely, just missed those in my find-and-replace. Should be updated now.